fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -55,6 +55,153 @@ object ImageCutout {
 
 sealed trait FaciaContent
 
+object FaciaContent {
+
+  def fold[T](fc: FaciaContent)(c: (CuratedContent) => T, scc: (SupportingCuratedContent) => T,
+              ls: (LinkSnap) => T, las: (LatestSnap) => T): T = fc match {
+    case curatedContent: CuratedContent => c(curatedContent)
+    case supportingCuratedContent: SupportingCuratedContent => scc(supportingCuratedContent)
+    case linkSnap: LinkSnap => ls(linkSnap)
+    case latestSnap: LatestSnap => las(latestSnap)
+  }
+
+  def id(fc: FaciaContent): String = fold(fc)(
+    curatedContent => curatedContent.content.id,
+    supportingCuratedContent => supportingCuratedContent.content.id,
+    linkSnap => linkSnap.id,
+    latestSnap => latestSnap.id)
+
+  def content(fc: FaciaContent): Option[Content] = fold(fc)(
+    curatedContent => Option(curatedContent.content),
+    supportingCuratedContent => Option(supportingCuratedContent.content),
+    linkSnap => None,
+    latestSnap => latestSnap.latestContent)
+
+  def maybeContent(fc: FaciaContent) = content(fc)
+
+  def supportingContent(fc: FaciaContent): List[FaciaContent] = fold(fc)(
+    curatedContent => curatedContent.supportingContent,
+    supportingCuratedContent => Nil,
+    linkSnap => Nil,
+    latestSnap => Nil)
+
+  def headline(fc: FaciaContent): String = fold(fc)(
+    curatedContent => curatedContent.headline,
+    supportingCuratedContent => supportingCuratedContent.headline,
+    linkSnap => linkSnap.headline.getOrElse("Missing Headline"),
```

Comment:
Why not an Option?

(I'd tend to push rendering something else to the most outer layer possible to give consumers of the client the choice.)